tests/unit/amo/components/TestHomepageShelves.js

```diff
@@ -102,26 +106,41 @@ describe(__filename, () => {
   });
 
   it.each([
-    [false, HOMESHELVES_ENDPOINT_COLLECTIONS],
-    [false, HOMESHELVES_ENDPOINT_SEARCH],
-    [true, HOMESHELVES_ENDPOINT_SEARCH_THEMES],
-  ])('passes isTheme as %s when endpoint is %s', (isTheme, endpoint) => {
-    const root = render({
-      shelves: [_createShelf({ endpoint })],
-    });
+    [false, HOMESHELVES_ENDPOINT_COLLECTIONS, 0],
```

Comment:
It feels a bit overboard with all the combinations, but I guess for now every path needs to be tested. Please open an issue to remove the code that bases this on `endpoint` after the server change lands, and then we can clean up these tests as well.